### ISSUE TITLE
A11y_Code.Visualstudio.Com(CELA-P1)_VS Code tips and tricks_Resize Text: Right side contents present in the page like 'In this Article','Basics','Command line','status bar' etc are not visible when the page is Zoomed to 200% 

### ISSUE THREAD

**Issue by saiprasanna93** [microsoft/vscode-docs#2650]
Check out Keros!- Identify accessibility bugs before check-in and make bug fixing faster and easier. 

**Environment Details:**
URL: code.visualstudio.com/docs/getstarted/tips-and-tricks
**Browser Details:** Microsoft Edge 44.17763.1.0,Google Chrome Version  74.0.3729.131 (Official Build) (64-bit)

**Note:**
Similar issue is observed in all the URLs mentioned below.
code.visualstudio.com/docs
code.visualstudio.com/docs/supporting/faq
code.visualstudio.com/docs/setup/linux
code.visualstudio.com/docs/getstarted/settings
code.visualstudio.com/docs/python/python-tutorial
code.visualstudio.com/updates/v1_30
code.visualstudio.com/docs/editor/whyvscode
code.visualstudio.com/docs/getstarted/tips-and-tricks
code.visualstudio.com/updates/v1_29
code.visualstudio.com/docs/editor/integrated-terminal
code.visualstudio.com/docs/editor/debugging
code.visualstudio.com/docs/editor/versioncontrol

**Repro Steps:​**
1)Hit URL:  code.visualstudio.com/docs/getstarted/tips-and-tricks and log in with appropriate credentials.​
2)Tab till Right side contents present in the page like 'In this Article', 'Basics', 'Command line', 'status bar' etc
3)Zoom the page to 200%
4)Verify whether Right side contents present in the page like 'In this Article', 'Basics', 'Command line', 'status bar' etc are visible or not when the page is zoomed to 200%

**Actual Result:**  
Right side contents present in the page like 'In this Article', 'Basics', 'Command line', 'status bar' etc are not visible when the page is Zoomed to 200% 

**Expected result:** 
Right side contents present in the page like 'In this Article', 'Basics', 'Command line', 'status bar' etc should be visible when the page is Zoomed to 200% 

**User Impact:** 
Low vision users and sighted users will face difficulty if content and controls are missing when zooming the page at 200%

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Comment by saiprasanna93** [comment]
[Bug_2650_Right side contents present in the page like 'In this Article','Basics','Command line','status bar' etc are not visible when the page is Zoomed to 200%.pptx](https://github.com/microsoft/vscode-docs/files/3161913/Bug_2650_Right.side.contents.present.in.the.page.like.In.this.Article.Basics.Command.line.status.bar.etc.are.not.visible.when.the.page.is.Zoomed.to.200.pptx)

**Comment by saiprasanna93** [comment]
![Bug_2650_Right side contents present in the page like 'Getting Started','VS Code in Action','Top Extensions','First steps'  etc are visible in normal 100% zooming](https://user-images.githubusercontent.com/50437422/57622540-7c7dfe00-75ab-11e9-9468-74a02f025af8.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_Right side contents present in the page like 'Getting Started','VS Code in Action','Top Extensions','First steps'  etc are not visible when the page is zoomed to 200%](https://user-images.githubusercontent.com/50437422/57622616-9e778080-75ab-11e9-83c7-2418e1a324d3.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_Right side contents present in the page like 'In this Article','Installations','Updates','Node js' etc are not visible when the page is Zoomed to 200%](https://user-images.githubusercontent.com/50437422/57622645-b18a5080-75ab-11e9-9bf6-3409c5469b00.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_Right side contents present in the page like 'In this Article','Installations','Updates','Node js'  etc are visible in normal 100% zooming](https://user-images.githubusercontent.com/50437422/57622671-bf3fd600-75ab-11e9-888f-61667e1efba6.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_Right side contents present in the page like 'In this Article','SCM providers','Gits upport','Commit' etc visible in normal 100% zooming](https://user-images.githubusercontent.com/50437422/57622696-cbc42e80-75ab-11e9-9e44-b7b354528e86.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_Right side contents present in the page like 'In this Article','SCM providers','Gits upport','Commit' etc are not visible in when the page is zoomed to 200%](https://user-images.githubusercontent.com/50437422/57622722-d54d9680-75ab-11e9-9ca6-b4c003d01972.png)

**Comment by auchenberg** [comment]
This is intentional and by design. We hide our sub menu navigation on smaller/zoomed screens to optimize the screen real-estate.

**Comment by saiprasanna93** [comment]
We discussed regarding this bug with our SME she suggested "There needs to be a way to get sub menu navigation back (whether through a hamburger menu, an accordion, etc) rather than just having it disappear" so If you need any clarification please reach out to the SME "Alexandria Reynolds" (v-alreyn@microsoft.com)

**Comment by gregvanl** [comment]
docs.microsoft.com injects a list of links at the top of a topic when zoomed.

![image](https://user-images.githubusercontent.com/12818376/59706417-4c7ed780-91b5-11e9-9ccf-e476d63a8bfe.png)

![image](https://user-images.githubusercontent.com/12818376/59706487-6c160000-91b5-11e9-8626-989320b02351.png)

**Comment by gregvanl** [comment]
The right navigation still shows for me on a 15" Surface Book 2 for both Edge and Chrome.

![image](https://user-images.githubusercontent.com/12818376/59954927-3afb3100-943c-11e9-9ec1-018fd4830fc8.png)

**Comment by auchenberg** [comment]
Same, can validate that the right-hand menu is visible in 200% in Chrome on MacOS.

<img width="1792" alt="Screen Shot 2019-06-23 at 12 25 25 PM" src="https://user-images.githubusercontent.com/173559/59980960-0e692580-95b2-11e9-9bd2-f25bdb5acce9.png">

Please advice on your used browser, OS and screen DPI. @saiprasanna93 saiprasanna93

**Comment by saiprasanna93** [comment]
@auchenberg 
Please find the details below
Microsoft Edge 44.18362.1.0,Google Chrome Version Version 75.0.3770.100
OS version-Microsoft Windows Version 1903 (OS Build 18362.175)
Display Resolution :1366X768

**Comment by octref** [comment]
I'm displaying them to the bottom.

![image](https://user-images.githubusercontent.com/4033249/60372209-b155e100-99b0-11e9-87c7-989938f4eec4.png)

**Comment by saiprasanna93** [comment]
Regressed the bug in the below environment
https://code.visualstudio.com/docs/getstarted/tips-and-tricks

Right side contents present in the page like 'In this Article', 'Basics', 'Command line', 'status bar' etc are now visible when the page is Zoomed to 200% now  hence closing this bug and added attachments for reference.

**Comment by saiprasanna93** [comment]
![Bug_2650_Issue resolved](https://user-images.githubusercontent.com/50437422/60483960-56283500-9cb5-11e9-8773-d036e41b5c84.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_Issue resolved now](https://user-images.githubusercontent.com/50437422/60483977-63452400-9cb5-11e9-9b04-a797aac70a17.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_issue fixed](https://user-images.githubusercontent.com/50437422/60483996-70621300-9cb5-11e9-894f-6ec927a317d9.png)

**Comment by saiprasanna93** [comment]
![Bug_2650_issue fixed now](https://user-images.githubusercontent.com/50437422/60484003-77892100-9cb5-11e9-9254-213b904fa49b.png)

**Comment by Roopesh-T** [comment]
**GitHubTag**: #closed; #regressed:05-31-23

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
